### ISSUE TITLE
Issue #128: Replace chosen.js with select2

### ISSUE THREAD

**Issue by sean-lewis** [sean-lewis/team_titan_test#14]
chosen.js has a long thread on the lack of accessibility (https://github.com/harvesthq/chosen/issues/264) and multiple other issues, a few with pull requests, but the maintainers have not adequately addressed the problems. Thus I'm going to replace it with select2: https://select2.org/

**Old Trello URL:** https://trello.com/c/nHEwNLbc/1214-issue-128-replace-chosenjs-with-select2

**Attachments:**
